packages/multichain/src/scope/authorization.ts

```diff
@@ -0,0 +1,44 @@
+import type { Json } from '@metamask/utils';
+
+import { normalizeAndMergeScopes } from './transform';
+import type {
+  ExternalScopesObject,
+  ExternalScopeString,
+  ScopesObject,
+} from './types';
+import { validateScopes } from './validation';
+
+export type Caip25Authorization = (
```

Comment:
Seems like an important type as well. Would it be worth documenting this, or at least linking to CAIP-25?

``` suggestion
/**
 * Represents the parameters of a [CAIP-25](https://chainagnostic.org/CAIPs/caip-25) request.
 */
export type Caip25Authorization = (